src/platforms/python/common/configuration/integrations/index.mdx

```diff
@@ -1,10 +1,34 @@
 ---
 title: Integrations
 description: "Sentry provides additional integrations designed to change configuration or add instrumentation to your application."
-sidebar_order: 700
+sidebar_order: 30
 ---
 
-Sentry provides a number of additional integrations which are designed to change configuration or add additional instrumentation to your application.
+Sentry provides a number of integrations which are designed to add automatic instrumentation to your application.
+
+Integrations can be configured in the call to `sentry_sdk.init()` like this:
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+
+sentry_sdk.init(
+    ...
+    integrations=[
+        DjangoIntegration(),
+        CeleryIntegration(),
+        RedisIntegration(),
+    ],
+)
+```
+
+**NEW:** The Sentry Python SDK also tries to automatically enable integrations that fit your Python modules you have installed. If you have Django installed the Sentry SDK will enable the `DjangoIntegration()` by default for you. No need for configuration at all!
```

Comment:
Adding markers like "new" on content means you then have to think about keeping track of when it's no longer new. Err on the side of making the content future-proof. Also, always avoid exclamation points. Generally we go for an informative, but casual tone.

```suggestion
The Sentry Python SDK also tries to automatically enable integrations that fit the Python modules you have installed. If you have Django installed, the Sentry SDK will enable the `DjangoIntegration()` by default for you without any further configuration.
```